src/Behavioral.Automation/Bindings/DropdownBinding.cs

```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
                 !behavior.Contains("not"), $"{wrapper.Caption} items are {items.Aggregate((x, y) => $"{x}, {y}")}");
         }
 
+        [Given("user selected \"(.*) in (.*)")]
```

Comment:
Should be 
` [Given("user selected \"(.*?)\" in (.*?)")]`

Reply:
Fixed